src/components/20-molecules/list/child.scss

```diff
@@ -0,0 +1,40 @@
+axa-list {
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+
+    & > li {
+        margin: 8px 0;
+        padding-left: 36px;
+        // This SVG renders a bullet point for unordered lists.
+        background-image: url('data:image/svg+xml;charset=UTF-8,<svg xmlns="http://www.w3.org/2000/svg" width="4" height="4" viewBox="0 0 4 4" fill="none"><rect width="4" height="4" rx="2" fill="black"/></svg>');
+        background-position: 0 10px;
+        background-repeat: no-repeat;
+    }
+    
+    &[variant="unstyled"] > li {
+        background-image: none;
+        background-position: 0;
+        padding-left: 0;
+    }
+    
+    &[variant="ordered"] > li {
+        background-image: none;
+        background-position: 0;
+        padding-left: 0;
+        counter-increment: number;
+        display: flex;
+
+        &::before {
+            display: block;
+            min-width: 34px;
```

Comment:
Magic number 34... why exact 34?

Reply:
So that it is aligned properly. Basically all numbers in here are magic numbers if you will. I don't have any variable name here that would "improve the situation", but maybe you have an idea.